examples/realworld/ext/auth/SignupPage.js

```diff
@@ -9,10 +9,10 @@ import { errorMessage } from '../utils.js'
 
 const SignupPage = () => {
   const history = useHistory()
-  const [username, setUsername] = useState()
-  const [email, setEmail] = useState()
-  const [password, setPassword] = useState()
-  const [submitError, setSubmitError] = useState()
+  const [username, setUsername] = useState('')
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [submitError, setSubmitError] = useState(null)
```

Comment:
`useState` returns `undefined`. Passing `undefined` as a `value` prop to an input tells it to be uncontrolled. Changing it to a defined value later tells it to be controlled. React doesn't like this. We can fix it by initializing controlled inputs to sensible default values.